pyproject.toml

```diff
@@ -44,6 +44,7 @@ num2tex = "^0.8"
 mkdocs-jupyter = "^0.17.3"
 
 [tool.poetry.extras]
+all = ["notebook", "matplotlib"]
 examples = ["notebook", "matplotlib"]
```

Comment:
With only one extra, I don't see the necessity for an `all` summary extra. Where would you see beneficial impact of having this?

Reply:
Did this for two reasons: 
1. I like this as a user, I just can install "all" without worrying which/how many extras are defined
2. We won't have to touch the ci/cd pipeline if new extras will be added

But I'm undecided, because 2.) probably won't happen anytime soon, and 1.) is more of a personal preference. 

Would you prefer it without the "all" extra?

Reply:
As discussed in person, let's remove the "all" extra until we need it.